### DISCUSSION TITLE
Build log redaction is too aggressive

### DISCUSSION THREAD

**Discussion by fernap3** [community/community#25852] [Actions]
<p>I understand that workflow logs redact some output by replacing that output with astrisks.  However it seems that it is redacting content that is not secret, for example the word “build” in my job named “Do the Linux build” is redacted:</p>
<p>Screenshot:</p>
<p><a href="https://drive.google.com/open?id=1A-Fyzw7aamJPzGWa8HowODOudUN3Ije7" rel="nofollow noopener">https://drive.google.com/open?id=1A-Fyzw7aamJPzGWa8HowODOudUN3Ije7</a></p>
<p>Anyone have insight into this?</p>
<p>Peter</p>

**Comment by ethomson** [comment, marked as answer]
<p>I’m afraid that I can’t see that screenshot.</p>
<p>Whenever I’ve run into this, it’s because I’ve set up a secret that contains that value and it’s being scrubbed according to the normal secrets-redaction rules.  For example, I’ve set up secrets like <code>REMOTE_USERNAME</code> and <code>REMOTE_PASSWORD</code>.  And when <code>REMOTE_USERNAME</code> is <code>build</code>, you’ll see that word redacted.</p>

**Comment by fernap3** [comment]
<p>Right on in this case. :slight_smile:</p>
<p>Thanks.</p>
